backend/src/main/resources/XSLT/html/literature.xslt

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+				xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+				xmlns:akn="http://docs.oasis-open.org/legaldocml/ns/akn/3.0"
+				xmlns:ris="http://ldml.neuris.de/literature/metadata/"
+				exclude-result-prefixes="akn ris">
+
+	<xsl:output method="html"/>
+
+	<xsl:strip-space elements="*"/>
+
+	<xsl:template match="akn:doc">
+		<html>
+			<head>
+				<title>
+					<xsl:value-of select="//akn:FRBRalias[@name='haupttitel']/@value"/>
+				</title>
+			</head>
+			<body>
+				<xsl:apply-templates/>
+			</body>
+		</html>
+	</xsl:template>
+
+	<!-- always render main title if it exists -->
+	<xsl:template match="akn:FRBRalias[@name='haupttitel']">
+		<h1><xsl:value-of select="@value"/></h1>
+	</xsl:template>
+
+	<!-- render documentary title differently based on main title existence -->
+	<xsl:template match="akn:FRBRalias[@name='dokumentarischerTitel']">
+		<xsl:variable name="haupttitelExists" select="../akn:FRBRalias[@name='haupttitel']/@value"/>
+		<xsl:choose>
+			<xsl:when test="$haupttitelExists">
+				<h3><xsl:value-of select="@value"/></h3>
+			</xsl:when>
+			<xsl:otherwise>
+				<h1><xsl:value-of select="@value"/></h1>
+			</xsl:otherwise>
+		</xsl:choose>
+	</xsl:template>
+
+	<!-- Outline -->
+	<xsl:template match="ris:gliederung">
+		<h2>Gliederung</h2>
+		<ul>
+			<xsl:apply-templates/>
+		</ul>
+	</xsl:template>
+
+	<xsl:template match="ris:gliederungEntry">
+		<li><xsl:value-of select="."/></li>
+	</xsl:template>
+
+	<!-- Short Report -->
+	<!-- if there is no short report the main body only contains an empty hcontainer -->
+	<xsl:template match="akn:mainBody">
+		<xsl:if test="not(akn:hcontainer)">
+			<h2>Kurzrefarat</h2>
+			<xsl:apply-templates />
+		</xsl:if>
+	</xsl:template>
+
+	<!--***************************************************************************************-->
+	<!--Html transformation-->
+	<!--***************************************************************************************-->
+
+	<!-- akn html elements with same name -->
+	<xsl:template match="akn:p|akn:div|akn:span|akn:sub|akn:sup">
+		<xsl:element name="{local-name()}">
+			<xsl:apply-templates/>
+		</xsl:element>
+	</xsl:template>
+
+	<xsl:template match="akn:br">
+		<br />
+	</xsl:template>
+
+	<xsl:template match="akn:a">
```

Comment:
This could be simplified to

    <xsl:template match="akn:a[@href]">
        <a href="{@href}">
            <xsl:apply-templates/>
        </a>
    </xsl:template>